platform/core/src/classes/CommandsManager.ts

```diff
@@ -173,7 +173,8 @@ export class CommandsManager {
    */
   public run(
     toRun: Command | Commands | Command[] | undefined,
-    options?: Record<string, unknown>
+    options?: Record<string, unknown>,
+    defaultCommand?: string
```

Comment:
Could you please provide a (use) case where/how `defaultCommand` is used? Thanks.